src/System.IO.Pipes/src/System.IO.Pipes.csproj

```diff
@@ -254,6 +254,12 @@
     <Reference Include="System.Threading.Overlapped" />
     <Reference Include="System.Threading.Tasks" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetsWindows)' == 'true'">
+    <Reference Include="System.IO.Pipes.AccessControl" />
```

Comment:
Lets avoid this reference we intentionally wanted System.IO.Pipes.AccessControl to be a higher level component.

Reply:
Yes I added it cause I missed it was OOB :) Removing and we can decide how much we want to move down to System.IO.Pipes.